docs: update readme (#17)
PandasWhoCode/update-custom-properties: README.md

# README for the project Update Custom Properties

**Update custom properties for all repos in an organization from a YAML file.**

This action uses the GitHub CLI to read values from `repo-properties.yaml` and update repos in the organization. This
can be useful for easily maintaining a single source of truth and a git-based history of updates to the
custom properties of the organization.

---

## 🚀 Getting Started

1. Add this GitHub Action to your workflow.

> 2. Run the workflow. The file `repo-properties.yaml` will automatically be parsed and all appropriate repo values

Reviewer: ```suggestion
2. Run the workflow. The file `repo-properties.yaml` will automatically be parsed (or generated) and all appropriate repo values
```

will be updated.

---

## Fine Grained Token Requirements

To run the action within your GitHub CI/CD pipeline you will need to create a
fine-grained token with the following permissions:

### Organization Permissions

> - Read and Write access to organization custom properties

Reviewer: ```suggestion
- Read access to organization custom properties
```

> - Read and Write access to organization custom properties

Reviewer: ```suggestion
- Read and Write access to repository custom properties
```

### Additional Information

- [GitHub API for custom property for an organization](https://docs.github.com/en/rest/orgs/custom-properties?apiVersion=2022-11-28#create-or-update-a-custom-property-for-an-organization)
- [Fine-grained personal access tokens](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-fine-grained-personal-access-token)
- The owner of the fine-grained token must have full administrative rights to the organization.

---

## 📦 Inputs

| Name                   | Description                                                       | Required | Default                |
|------------------------|-------------------------------------------------------------------|----------|------------------------|

> | `token`                | GitHub Personal Access Token (Fine-Grained with `Read` org scope) | ✅ Yes    | —                      |

Reviewer: ```suggestion
| `token`                | GitHub Personal Access Token (Fine-Grained with Repository properties: `Read and Write` scope) | ✅ Yes    | —                      |
```
